yarn.lock

```diff
@@ -2,17 +2,30 @@
 # yarn lockfile v1
```

Comment:
Upgrade this repo to use `PNPM` instead

Reply:
Handled in: https://github.com/journeyapps/pdf-reports/pull/16